src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSSurfaceNamer.java

```diff
@@ -576,6 +576,7 @@ public String getFieldAccessorName(FieldModel field) {
 
   @Override
   public List<String> getPrintSpecs(String spec, List<String> args) {
+    spec = spec.replace("\\", "\\\\").replace("\t", "\\t").replace("\n", "\\n");
```

Comment:
Since we output strings within backticks, we should escape the backticks, too.

Also, some of our strings (eg defaults) have single quotes, not backticks. So in that case we should escape _those_.